contributors/devel/release.md

```diff
@@ -0,0 +1,246 @@
+# Targeting Features, Issues and PRs to Release Milestones
+
+This document is focused on Kubernetes developers and contributors
+who need to create a feature, issue, or pull request which targets a specific
+release milestone.
+
+-   [Definitions](#definitions)
+-   [The Release Cycle](#the-release-cycle)
+-   [Removal Of Items From The Milestone](#removal-of-items-from-the-milestone)
+-   [Adding An Item To The Milestone](#adding-an-item-to-the-milestone)
+    -   [Milestone Maintainers](#milestone-maintainers)
+    -   [Feature additions](#feature-additions)
+    -   [Issue additions](#issue-additions)
+    -   [PR Additions](#pr-additions)
+    -   [Other Required Labels](#other-required-labels)
+        -   [SIG Owner Label](#sig-owner-label)
+        -   [Priority Label](#priority-label)
+        -   [Issue Kind Label](#issue-kind-label)
+
+The process for shepherding features, issues, and pull requests
+into a Kubernetes release spans multiple stakeholders:
+* the feature, issue, or pull request owner
+* SIG leadership
+* the release team
+Information on workflows and interactions are described below.
+
+As the owner of a feature, issue, or pull request (PR), it is your
```

Comment:
What's my tl;dr here?  I appreciate there may be a lot of context below, but you're going to lose me in all of those paragraphs

Reply:
I think the tl;dr I'm looking for is something like "what do I have to do to get my code into the upcoming release"
- Initially you must frob the whatsit
- During code freeze you must flip the doodad
- After code freeze you must perform an interpretive dance for the following audience

There also seems to be an undefined implication that being "in the milestone" is the same that as being "in the release".  That's not really true today, there are tons of PR's being merged right that have no milestone attached.

Reply:
this state of things does beg the question on what is _required_ for a milestone/why we have a milestone process at all.

Reply:
added TL;DR table